chart/inlets-operator/values.yaml

```diff
@@ -26,6 +26,12 @@ accessKeyFile: "/var/secrets/inlets/inlets-access-key"
 # that requires an access key and secret key, such as EC2.
 secretKeyFile: ""
 
+# The VPC ID to create the exit-server in (EC2)
```

Comment:
I'm wondering if it'd be clearer / cleaner to namespace this to:

```
ec2:
  vpcId:
  subnetId
```

It depends on whether we'll need this for other providers

Reply:
Leaving for now. Happy to address in a subsequent PR if need be.